src/frontend/types/state-types.ts

```diff
@@ -44,4 +44,5 @@ export interface State {
   showFieldSet: boolean
   documentTypes: string[]
   sorting: Partial<Record<Field, SortAction>>
+  CSVDownloadType: string
```

Comment:
question, should this use the enum? `CSVDownloadType: CSVDownloadType`?  If so, the enum definition would belong in this file too.